src/app/core/services/config/config.service.ts

```diff
@@ -29,7 +29,7 @@ import { SubjectConfigService } from './subject-config.service'
 
 @Injectable()
 export class ConfigService {
-  ATTRIBUTE_KEY_PREFIX = 'att-'
+  ATTRIBUTE_KEY_PREFIX = 'att_'
```

Comment:
The attribute name itself contains hyphens (like `human-readable-id`), those will also need to be sanitised. You can do a regex replace of unsupported chars with `_`.

Reply:
Ah that's true. I'll make the changes.